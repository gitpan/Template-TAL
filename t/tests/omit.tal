<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body>
    <p tal:omit-tag="string:1">no tag</p>
    <p tal:omit-tag="string:0">has tag</p>
    <p tal:omit-tag="string:1"><span tal:replace="string:still no tag"/></p>
  </body>
</html>
