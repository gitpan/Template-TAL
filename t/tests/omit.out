<html>
  <body>
    no tag
    <p>has tag</p>
    still no tag
  </body>
</html>
